cg/utils/files.py

```diff
@@ -1 +1,22 @@
 """Some helper functions for working with files"""
+from pathlib import Path
+from typing import Optional
+import os
+
+
+def get_file_in_directory(directory: Path, file_name: str) -> Path:
+    """Get a file in a directory and subdirectories.
+    Raises:
+        FileNotFoundError: If the file is not found in the directory.
+        FileNotFoundError: If the directory does not exist.
+    """
+    if not directory.is_dir() or not directory.exists():
+        raise FileNotFoundError(f"Directory {directory} does not exist")
```

Comment:
I agree that the file is not found but is FileNotFound also intended to be used for missing directories?

Reply:
No such file OR directory is the error message in python